### PR TITLE
Implement activity tracker

### PR DESCRIPTION
**Pull request by mmorhun**
Signed-off-by: Mykola Morhun <mmorhun@redhat.com>

Actual implementation of the activity tracker and its docs.
Tested on docker and openshift infrastructures.

### CODE REVIEW

**Review by benoitf — APPROVED** [review]
I think it's the last extension I approved.
it should be a plug-in.

**Review by AndrienkoAleksandr — APPROVED** [review]

**Review by akurinnoy — APPROVED** [review]

**Review by vitaliy-guliy — APPROVED** [review]

**Comment by sunix** [comment]
why GPL-2.0 WITH Classpath-exception-2.0 ?

**Comment by mmorhun** [comment]
@sunix because Theia has this license.

File: che-theia-activity-tracker/package.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+  "name": "che-theia-activity-tracker",
```
Comment (sunix) [inline che-theia-activity-tracker/package.json]: shouldn't it be `@eclipse-che/theia-activity-tracker` ?

Comment (mmorhun) [reply che-theia-activity-tracker/package.json]: Yes, thanks, will fix

**Comment by benoitf** [comment]
BTW for the licence, EPL 2.0 (no need of exception) is fine, it is not contributed to theia

**Review by ashumilova — APPROVED** [review]

**Pull request merged**